admin/acceptance.py

```diff
@@ -269,6 +271,12 @@ def start_nodes(self, reactor):
                            variants=self.variants)
             for node in self.nodes
         ])
+        if self.volume_backend == VolumeBackend.zfs:
```

Comment:
I don't want to do dispatch on this yet, until we see what other backends might need.